src/components/Main/Main.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import Controls from '../Controls/Controls.js';
+import Player from '../Player/Player.js';
+import Slogan from '../Slogan/Slogan.js';
+import Stats from '../Stats/Stats.js';
+
+import './Main.css';
+
+export default function Main() {
+  const [helmet, setHelmet] = useState('blue');
+  const [jersey, setJersey] = useState('home');
+  const [pants, setPants] = useState('white');
+  const [helmetCount, setHelmetCount] = useState(0);
+  const [jerseyCount, setJerseyCount] = useState(0);
+  const [pantsCount, setPantsCount] = useState(0);
+  const [slogans, setSlogans] = useState(['Bear Down!']);
+
+  const handleChange = (type, value) => {
+    if (type === 'helmet'){
+      setHelmet(value);
+      setHelmetCount(helmetCount + 1);
```

Comment:
Any time you are going to update state based on the value of the previous state you should pass callback function to `setState`. In this case your `setHelmetCount(helmetCount + 1)` should be changed to `setHelmentCount((prevState) => prevState + 1)`

The React docs mention it [here](https://reactjs.org/docs/hooks-reference.html#functional-updates)

Reply:
Awesome, thanks for the feedback Vic!